### DISCUSSION TITLE
Delete future branch?

### DISCUSSION THREAD

**Discussion by bryevdv** [bokeh/bokeh#12334] [Internals and design (Q&A)]
@mattpap unless you object, I would delete the `future` branch. We have never really used it and it seems like trying to use it would just be a headache. If the uncommon situation ever does arise, we can just create the appropriate landing branch, if necessary, or else let the PR stay open an extended period.

**Comment by mattpap** [comment]
`future` was early stage `branch-3.0`. There is no need for it anymore and it can be safely deleted.
